source/features/expand-urls.js

```diff
@@ -0,0 +1,88 @@
+import ky from 'ky';
+
+const refinedTwitterClass = 'refinedTwitterExpandedUrl';
+
+async function getUnshortenUrl(shortURL) {
+	const json = await ky.get('https://linkpeelr.appspot.com/api?action=peel_all&url='
+	+ shortURL + '&where=twitter.com&version=2.0.3').json();
+
+	return manageAPIResponse(json, shortURL);
+}
+
+// linkpeelr API
+function parseAPIResponse(apiResponse) {
+	return apiResponse[1];
+}
+
+// linkpeelr API
+function isAPIResponseValid(apiResponse) {
+	return apiResponse[0] === 301;
+}
+
+function manageAPIResponse(apiResponse, shortURL) {
+	if (isAPIResponseValid(apiResponse)) {
+		const apiURL = parseAPIResponse(apiResponse);
+		return apiURL;
+	} else {
+		return shortURL;
+	}
+}
+
+/**
+ * remove http(s) and www from a URL
+ **/
+function removeHTTPAndWWW(string){
+  return string.replace('https://', '').replace('http://', '').replace('www.', '');
+}
+
+function setURL(url, htmlElement) {
+	htmlElement.setAttribute('href', url);
+}
+
+function removeUTMs(url) {
+	const parsedURL = new URL(url);
+	const urlWithOutUTMs = parsedURL.href.replace(/[?&#]utm_.*/, '');
+
+	return urlWithOutUTMs;
+}
+
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+// Example: https://twitter.com/polomarcus/status/955649699477950464
+async function twitterCardURLs() {
+	await sleep(2000); //iframes takes longer to load
```

Comment:
Annoying... There is small lag when loading a Tweet card forcing me to use a `sleep` :confused: